### DISCUSSION TITLE
before:absolute class not generated

### DISCUSSION THREAD

**Discussion by ilhamfu** [tailwindlabs/tailwindcss#9270] [Help]
Tailwindcss : 3.1.8
Node Version : 18.7.0
NextJs : 12.5.5

before:absolute class didn't get generated when using with npm run dev, and it only happen when i use npm run dev. if i try to generate css file using tailwindcss CLI it get generated

**Comment by smithliam** [comment]
Look at the docs for NextJs: https://tailwindcss.com/docs/guides/nextjs
You need to do some stuff, so your framework can generate the classes... Tailwind CLI works without the need for NextJs, so that's why it works...
